lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java

```diff
@@ -80,10 +78,14 @@ public LightblueNotificationRepository(LightblueClient lightblue, String[] entit
             DataBulkRequest updateEntities = new DataBulkRequest();
```

Comment:
I see. Eventually though I will be persisting different status updates based on the entity (processing or failed). I could turn that into two requests in a bulk request, I think though. Will open separate issue.